azext_iot/iothub/commands_state.py

```diff
@@ -0,0 +1,33 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from azext_iot.iothub.providers.job import JobProvider
+from azext_iot.iothub.providers.state import StateProvider
+
+import time
+
+logger = get_logger(__name__)
+
+
+def state_export(cmd, filename, hub_name=None, resource_group_name=None, login=None, auth_type_dataplane=None):
+    b = time.perf_counter()
```

Comment:
make sure to remove this in the final thing - but having the times will be interesting (and good data points for your presentation)